pkg/tc/tc.go

```diff
@@ -75,12 +75,16 @@ func enableQdisc(link netlink.Link) bool {
 
 }
 
-func mismatchedInterfacePrefix(interfaceName string, interfacePrefix string) error {
-	if !strings.HasPrefix(interfaceName, interfacePrefix) {
-		log.Errorf("expected prefix - %s but got %s", interfacePrefix, interfaceName)
-		return errors.New("Mismatched initialized prefix name and passed interface name")
+func mismatchedInterfacePrefix(interfaceName string, interfacePrefix []string) error {
```

Comment:
You will have update this - https://github.com/aws/aws-ebpf-sdk-go/blob/main/pkg/tc/tc_test.go#L99